restli-tools/src/main/resources/apiVmTemplates/rest.partial_update.vm

```diff
@@ -0,0 +1,19 @@
+#*
+   Copyright (c) 2021 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*#
+#parse("apiVmTemplates/rest.partial_update_no_return.vm")
```

Comment:
What template based codegen engine are you using? Not Rythm?

Reply:
This is based on velocity.